collections/_posts/2022-11-23-Multi Channel Stochastic Variational Inference.md

```diff
@@ -0,0 +1,107 @@
+    title:   "Multi-Channel Stochastic Variational Inference for the Joint Analysis of Heterogeneous Biomedical Data in Alzheimer's Disease"
+    venue:   "arXiv 2018"
+pdf: "https://arxiv.org/pdf/1808.03662.pdf"
+---
+
+
+
+# Notes
+
+* Unsupervised method.
+* **Gaussian linear case** (for validation purposes)
+* An other paper based on deep learning methods was done in 2019 by the same authors : [Sparse Multi Channal Variationnal Auto Encoder](http://proceedings.mlr.press/v97/antelmi19a/antelmi19a.pdf).
+* A [Github repository](https://gitlab.inria.fr/epione_ML/mcvae) is available  for the 2019 paper.
+
+* Link to the [VAE tutorial](https://creatis-myriad.github.io/tutorials/2022-09-12-tutorial-vae.html) 
+
+
+# Highlights
+
+* The aim of this paper is to take into account several inputs (channels, potentially of different types) for a better joint analysis.
+* They proposed a novel Multi-channel stochastic generative model.
+* Their proposed method can have applications for a generic data fusion technique.
+
+
+# Introduction
+
+When a patient suffer from Alzheimer’s disease, physicians use lots of various sources of informations. However, simple univariate correlation analyses are limited in modeling power. To overcome the limitations of mass-univariate analysis, methods such as PLS, RRR or CCA and their variants (non-linearity, multi-channels) were used in biomedical research. The main issue is that those methods are not generative.
+The authors proposed a novel multi-channel stochastic generative model for the joint analysis of multi-channel heterogeneous data.
+
+
+# Methods
+
+Let $$ \textbf{x}  = \{\textbf{x}_c\}_{c=1}^{C} $$ be a single observation of a set of C channels, where each $$ \textbf{x}_c ∈ \mathbb{R}^{d_c} $$ is a $$ d_c $$-dimensional vector.
+
+* $$ p(\textbf{z}) $$ : prior
+* $$ p(\textbf{x}_c \vert \textbf{z}, \pmb{θ}_c) $$ : likelihood distribution
+* $$ q(\textbf{z} \vert \textbf{x}_c, \pmb{\phi}_c) $$ : probability density function
+
+They aim at minimiing :
+
+$$ \underset{q∈Q}{argmin} \: \mathbb{E}_c[D_{KL} (q (\textbf{z} \vert \textbf{x}_c, \pmb{\phi}_c) \parallel p (\textbf{z} \vert \textbf{x}_1, . . . , \textbf{x}_C , \pmb{θ}_1, . . . , \pmb{θ}_C ) )] $$
+
+**Hypothesis :** Every channel is conditionally independent from all the others given z (it will allow to factorize the data likelihood, see supplementary material in the paper for more details). 
+
+After developping the equation from above they have :
+
+$$ \mathscr{L}(\textbf{θ}, \pmb{\phi}, \textbf{x}) = \mathbb{E}_c \left[ \mathbb{E}_{q(\textbf{z} \vert \textbf{x}_c )} \left[  \sum_{i=1}^{C} ln \: p(\textbf{x}_i \vert \textbf{z}, \textbf{θ}_c) \right] − D_{KL} (q (\textbf{z} \vert \textbf{x}_c, \pmb{\phi}_c) \parallel p (\textbf{z}) ) \right] $$
+
+
+A figure used in the [Github repository](https://gitlab.inria.fr/epione_ML/mcvae) to understand the global idea :
+
+![](/collections/images/MCSVI/Multi Latent Space.jpg)
```

Comment:
I would recommend to remove spaces from the filenames of the images (and of the post itself). It seems to be working without problem, but I've had enough problems with some tools in Linux when filenames have spaces in them to know to be cautious about this :wink: